datafusion/core/src/execution/context.rs

```diff
@@ -2058,6 +2075,33 @@ fn create_dialect_from_str(dialect_name: &str) -> Result<Box<dyn Dialect>> {
     }
 }
 
+/// Deserializer registry for extensions like [UserDefinedLogicalNode].
+pub trait ExtensionDeserializer: Send + Sync {
```

Comment:
what do you think about putting this trait in `datafusion/execution/src/registry.rs` alongside 

```
25: pub trait FunctionRegistry 
```

Which serves a similar purpose for user defined functions?

Reply:
Great reference, I'll move it there